cmd/leveldbbench/usage.md

```diff
@@ -0,0 +1,115 @@
+This command is meant to give us a sense of the system level
+performance for leveldb:
+
+```bash
+go run main.go leveldbbench --degree-of-parallelism 2 | jq '.' > result.json
+```
+
+In many cases, we'll want to emulate the performance characteristics
+of `bor` or `geth`. This is the basic IO pattern when `bor` is in sync:
+
+```text
+Process Name = bor
+     Kbytes              : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 10239    |****************                        |
+         8 -> 15         : 25370    |****************************************|
+        16 -> 31         : 7082     |***********                             |
+        32 -> 63         : 1241     |*                                       |
+        64 -> 127        : 58       |                                        |
+       128 -> 255        : 11       |                                        |
+```
```

Comment:
nice so this is the baseline for a synced bor node

Reply:
Yeah. What's weird is this is quite different than the underlying level db key sizes. There are several layers of abstraction between leveldb api and underlying disk IOPs, so I should t be surprised.